### PR TITLE
Base: Add manpage for pkg

### PR DESCRIPTION
**Pull request by cubiclove**

### CODE REVIEW

File: Base/usr/share/man/man1/pkg.md
Comment (supercomputer7) [inline Base/usr/share/man/man1/pkg.md:13]: `https://ports.serenityos.net` is an official website of the SerenityOS project. I am not against adding it, but maybe we should wait for @awesomekling to approve on this.

**Review by supercomputer7 — APPROVED** [review]
I think this looks very nice :)
Let's see what other maintainers & reviewers have to say about the unofficial SerenityOS Ports website

**Comment by awesomekling** [comment]
Seems fine!

**Pull request merged**
